apps/src/templates/instructions/TeacherFeedback.jsx

```diff
@@ -161,6 +171,68 @@ export class TeacherFeedback extends Component {
     );
   }
 
+  renderFeedbackTeacherHeader() {
+    // Pilots which the user is enrolled in (such as keep working experiment) are stored on
+    // window.appOptions.experiments, which is queried by experiments.js
+    const keepWorkingEnabled = experiments.isEnabled(keepWorkingExperiment);
+    const latestFeedback = this.state.latestFeedback;
+
+    return (
+      <div style={styles.header}>
```

Comment:
NIT: could be nice to factor out a `renderHeader(content)` function with the `styles.header` and `h1` containers

Reply:
I'm going to leave this one as-is, for readability